config/environments/production.rb

```diff
@@ -56,7 +56,7 @@
   config.log_tags = [ :request_id ]
 
   # Use a different cache store in production.
-  config.cache_store = :redis_cache_store
+  config.cache_store = :redis_cache_store, { url: ENV.fetch("SIDEKIQ_REDIS_URL") { "redis://localhost:6379/1" } }
```

Comment:
It seems to me like we should not use `SIDEKIQ_REDIS_URL` as the name for a redis instance that is used for more than Sidekiq.   For example in Exhibits we use a different Redis database for the cache than for sidekiq. https://github.com/sul-dlss/shared_configs/blob/exhibits-prod/config/initializers/sidekiq.rb#L9 (note the `/1` vs `/2` difference)